src/DesignQuery/scripts/cpp_script_for_sdt_generation/output_sdtgen_cpp_nlohman_lib_v5.sdt

```diff
@@ -0,0 +1,154 @@
+/*
+     *
+     *Copyright (c) 2023 Rapid Silicon
```

Comment:
Should these data be on open-source?

Reply:
Rahul would be able to answer that. He did initially say that it should be okay.